f2/src/forms/__tests__/HowDidItStartForm.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```

Comment:
Isn't this test already in the code?

Reply:
Yes it is! Oups, not sure how it ended up here again :P